apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/RDATestUtils.java

```diff
@@ -168,28 +215,76 @@ private PreAdjFissClaim fissTestDataB() {
             .mbi("123456MBI")
             .mbiHash("a7f8e93f09")
             .fedTaxNumber("abc124")
+            .lobCd("k")
             .lastUpdated(Instant.ofEpochMilli(5000))
             .stmtCovFromDate(LocalDate.ofEpochDay(211))
             .stmtCovToDate(LocalDate.ofEpochDay(211))
+            .servTypeCd("A")
+            .freqCd("C")
             .build();
 
-    PreAdjFissProcCode code =
-        PreAdjFissProcCode.builder()
-            .dcn("123457")
-            .priority((short) 0)
-            .procCode("CODEABD")
-            .procFlag("FLAC")
-            .procDate(LocalDate.ofEpochDay(211))
-            .lastUpdated(Instant.ofEpochMilli(5000))
-            .build();
+    Set<PreAdjFissProcCode> procCodes =
+        Set.of(
+            PreAdjFissProcCode.builder()
+                .dcn("123457")
+                .priority((short) 0)
+                .procCode("CODEABD")
+                .procFlag("FLAC")
+                .procDate(LocalDate.ofEpochDay(211))
+                .lastUpdated(Instant.ofEpochMilli(5000))
+                .build());
+
+    Set<PreAdjFissDiagnosisCode> diagnosisCodes =
+        Set.of(
+            PreAdjFissDiagnosisCode.builder()
+                .dcn("123457")
+                .priority((short) 0)
+                .diagCd2("princcc")
+                .diagPoaInd("Y")
+                .build(),
+            PreAdjFissDiagnosisCode.builder()
+                .dcn("123457")
+                .priority((short) 1)
+                .diagCd2("other2")
+                .diagPoaInd("w")
+                .build(),
+            PreAdjFissDiagnosisCode.builder()
+                .dcn("123457")
+                .priority((short) 2)
+                .diagCd2("admitcc")
+                .diagPoaInd("1")
+                .build());
+
+    Set<PreAdjFissPayer> payers =
+        Set.of(
+            PreAdjFissPayer.builder()
+                .dcn("123457")
+                .priority((short) 0)
+                .beneFirstName("alice")
+                .beneMidInit("r")
+                .beneLastName("smith")
+                .beneSex("f")
+                .beneDob(LocalDate.of(1981, 8, 13))
+                .payerType(PreAdjFissPayer.PayerType.BeneZ)
+                .payersName("MEDICARE")
+                .build(),
+            PreAdjFissPayer.builder()
+                .dcn("123457")
+                .priority((short) 1)
+                .insuredName("SMITH  ALICE  R")
```

Comment:
Is there supposed to be two spaces between the names?

Reply:
That I'm not sure of, none of our logic cares, but I saw they had what seemed to be two spaces in the test env database.